pkg/minikube/cluster/cluster.go

```diff
@@ -393,6 +393,7 @@ func getDriverOptions(config MachineConfig) (drivers.DriverOptions, error) {
 
 		return createDriverOptions(d, machineConfigOptions)
 	case "vmwarefusion":
+		fmt.Println("VMWare fusion driver is deprecating soon. Please consider using other drivers.")
```

Comment:
VMware Fusion is improperly capitalized.

Personally, I would also prefer "will be deprecated" to "is deprecating".